db/migrate/20190912143132_add_release_field_to_os.rb

```diff
@@ -0,0 +1,40 @@
+class AddReleaseFieldToOs < ActiveRecord::Migration[5.2]
+  class Operatingsystem < ApplicationRecord
```

Comment:
If we prefer to keep this class, could we name it something like `OperatingsystemStubbed` and set tablename? It would be more clear we use a fake model.